guides/source/active_record_basics.md

```diff
@@ -223,10 +224,10 @@ class CreateBooks < ActiveRecord::Migration
 end
 ```
 
-The SQL as well as the migration above declare a table with three columns: `id`,
-`title`, and `author`. Each row of this table can be represented by an instance
-of the `Book` class with the same three attributes: `id`, `title`, and `author`.
-You can access a book's attributes like this:
+That migration creates columns `id`, `title`, `author`, `created_at` and
+`updated_at`. Each row of this table can be represented by an instance of the
+`Book` class with the same attributes: `id`, `title`, `author`, `created_at`,
+and `updated_at`. You can access a book's attributes like this:
```

Comment:
I like this. Listing all columns and attributes is more complete and clear. I went back and forth about how much to explain about how the `created_at`, `updated_at` and `id` are generated vs. the other columns.